components/Storyblok/SbStoryFilterPage/SbStoryFilterPage.tsx

```diff
@@ -35,18 +51,86 @@ export const SbStoryFilterPage = ({
   blok,
   name,
   slug,
+  storyList,
 }: SbStoryFilterPageProps) => {
+  /**
+   * Extract uuid's from the featured stories into a Set.
+   * The reason for using a Set in this case is its O(1) average time complexity for lookups,
+   * which is more efficient than using an array for the same purpose.
+   */
+  const featuredStoryUUIDSet = new Set(featuredStories?.map(item => item.storyPicker.uuid));
+
+  /**
+   * Filter storyList (list of all stories with taxonomy that matches the slug of the page)
+   * to exclude items with uuid that are in the featuredStoryUUIDArray,
+   * ie, we remove stories that are already added as featured stories cards.
+   */
+  const filteredStoryList = storyList.filter(item => !featuredStoryUUIDSet.has(item.uuid));
```

Comment:
This is how I exclude the featured stories from the stories rendered in the list.